### DISCUSSION TITLE
What is the grey box in the upper right (Windows-Version)?

### DISCUSSION THREAD

**Discussion by tih2010** [gitextensions/gitextensions#11082] [General] [answered]
Sorry, may be a dumb question but in our company no one knows:

What is the grey box in the upper right? We never saw it filled with anything.
And can we get rid of it?

![image](https://github.com/gitextensions/gitextensions/assets/2473190/23071785-6ec7-4a01-a7cf-c1f640871bc0)

Best,
    Wolfram

**Comment by gerhardol** [comment, marked as answer]
Fixed in https://github.com/gitextensions/gitextensions/pull/10968

Just toggle the positioning of commit info
![image](https://github.com/gitextensions/gitextensions/assets/6248932/f203e67d-081e-4c6e-8c70-39c3c5053782)

**Reply by tih2010** [reply]
Great! Thanks a lot!
